src/main/java/com/project/model/Application.java

```diff
@@ -21,53 +22,53 @@
 
     @NotBlank(message = "Vardas negali būti tuščias")
     @Size(max = 255, message = "Vardas negali būti ilgesnis nei 255 simboliai")
-    @Column(name="full_name")
+    @Column(name = "full_name")
     private String name;
 
     @NotBlank(message = "Telefono numeris negali būti tuščias")
     @Size(max = 20, message = "Telefono numeris negali būti ilgesnis nei 20 simbolių")
-    @Column(name="phone_number")
+    @Column(name = "phone_number")
     private String phone_number;
```

Comment:
Please change variable name to `phoneNumber`.

Reply:
changed the names of all variables in Application class to camel case